docs-client/webpack.config.ts

```diff
@@ -47,6 +47,7 @@ const config: Configuration = {
                   {
                     modules: false,
                     useBuiltIns: 'entry',
+                    corejs: 3,
                     targets: { browsers: ['last 2 versions'] },
```

Comment:
Thanks - while we're here, how about updating this browserlist query to this one

https://browserl.ist/?q=>1%25%2C+not+ie+11%2C+not+op_mini+all

I found that I've been using `last 2 versions` which is fairly outdated practice, it includes all browsers, including IE10 which has no support for ES6 at all and everything gets polyfilled! I think our jar will get quite a bit smaller.

Reply:
Updated. Down-sized from 1.22 MiB to 1.15 MiB.